runtime/databricks/automl_runtime/forecast/utils.py

```diff
@@ -14,14 +14,77 @@
 # limitations under the License.
 #
 import logging
-from typing import List, Optional
+from typing import Dict, List, Optional, Tuple, Union
 from databricks.automl_runtime.forecast import DATE_OFFSET_KEYWORD_MAP,\
     QUATERLY_OFFSET_ALIAS, NON_DAILY_OFFSET_ALIAS, OFFSET_ALIAS_MAP, PERIOD_ALIAS_MAP
 
 import pandas as pd
 
 _logger = logging.getLogger(__name__)
 
+def make_future_dataframe(
+        start_time: Union[pd.Timestamp, Dict[Tuple, pd.Timestamp]],
+        end_time: Union[pd.Timestamp, Dict[Tuple, pd.Timestamp]],
+        horizon: int,
+        frequency: str,
+        include_history: bool = True,
+        groups: List[Tuple] = None,
+        group_names: List[str] = None,
+) -> pd.DataFrame:
+    if groups is None:
+        return make_single_future_dataframe(start_time, end_time, horizon, frequency)
+
+    future_df_list = []
+    for group in groups:
+        if type(start_time) is dict:
+            group_start_time = start_time[group]
+        else:
+            group_start_time = start_time
```

Comment:
In which cases we'll have a single timestamp as `start_time` when we have multiple series? I thought we always use a dict for multi series cases.

Reply:
We don't have this use case. Add this part only to avoid unknown error when user try to call this function.